### DISCUSSION TITLE
game crash

### DISCUSSION THREAD

**Discussion by 3lovetobepink** [FabricMC/fabric-api#1514] [Player support] [answered]
when I try to load fabric for 1.17 it says my game crashed here's the log 
[latest.log](https://github.com/FabricMC/fabric/files/6697703/latest.log)

**Comment by haykam821** [comment, marked as answer]
[Terrestria](https://github.com/TerraformersMC/Terrestria) and [Traverse](https://github.com/TerraformersMC/Traverse) are not available on 1.17.
